notebook3/Test notebook 3 delta.ipynb

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
<p>Why are we importing this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kmh4321/test_WS_integration/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>